ts/features/idpay/onboarding/screens/InitiativeSelfDeclarationsScreen.tsx

```diff
@@ -0,0 +1,79 @@
+import { useActor } from "@xstate/react";
+import React from "react";
+import { View } from "native-base";
+import { SafeAreaView } from "react-native";
+import { ScrollView } from "react-native-gesture-handler";
+import { H1 } from "../../../../components/core/typography/H1";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import LoadingSpinnerOverlay from "../../../../components/LoadingSpinnerOverlay";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import { useOnboardingMachineService } from "../xstate/provider";
+import { Body } from "../../../../components/core/typography/Body";
+import { Link } from "../../../../components/core/typography/Link";
+import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import ListItemComponent from "../../../../components/screens/ListItemComponent";
+import { LOADING_TAG } from "../../../../utils/xstate";
+
+
+const InitiativeSelfDeclarationsScreen = () => {
+  const machine = useOnboardingMachineService();
+  const [state] = useActor(machine);
+
+  const isLoading = state.tags.has(LOADING_TAG);
+
+  return (
+    <BaseScreenComponent
+      headerTitle="Adesione all'iniziativa"
+      goBack={true}
+      contextualHelp={emptyContextualHelp}
+    >
+      <LoadingSpinnerOverlay isLoading={isLoading}>
+        <SafeAreaView style={IOStyles.flex}>
+          <ScrollView style={IOStyles.flex}>
+            <View style={IOStyles.horizontalContentPadding}>
+              <H1>Per aderire, dichiari di:</H1>
+              <View spacer={true} />
+              <Body>L’autodichiarazione è resa ai sensi del</Body>
+              <Link>Dpr 28 dicembre 2000 n. 445 art 46 e 47</Link>
+              <View spacer={true} large={true} />
+              <ListItemComponent
```

Comment:
Why we are not taking the data from the context?

Reply:
There is no info on auto declarations in the context yet. This is just a placeholder text.